RELEASE/scripts/autoscend/combat/auto_combat_ed.ash

```diff
@@ -375,7 +375,7 @@ string auto_edCombatHandler(int round, monster enemy, string text)
 		combat_status_add("banishercheck");
 	}
 
-	if (!combat_status_check("replacercheck") && canReplace(enemy) && auto_wantToReplace(enemy, my_location()))
+	if (!combat_status_check("replacercheck") && auto_wantToReplace(enemy, my_location()))
```

Comment:
Thinking out loud. Don't need to add support for parsing funkslinging like you did in the general combat code because Ed can't throw 2 items in a single round of combat. True fact?

Reply:
Never ran Ed but looking through the wiki that makes sense to me since you lose your skills and there is no funkslinging equivalent in the skill tree